packages/core/src/actions/send.ts

```diff
@@ -95,6 +95,11 @@ export function send<
       const delaysMap = state.machine.options.delays;
 
       // TODO: helper function for resolving Expr
+      if (typeof eventOrExpr === 'string') {
+        throw new Error(
+          `Only event objects may be sent to actors; use .send({ type: "${eventOrExpr}" }) instead`
+        );
```

Comment:
```suggestion
        throw new Error(
          `Only event objects may be used with sendTo; use sendTo({ type: "${eventOrExpr}" }) instead`
        );
```

Reply:
Note that I used here `sendTo` instead of `send` - despite the fact that this is in the `send`'s implementation. In a follow-up PR we should remove `send` altogether and just inline its implementation into `sendTo`.